src/Sentry/Platforms/Android/SentrySdk.cs

```diff
@@ -50,120 +50,130 @@ private static void InitSentryAndroidSdk(SentryOptions options)
         // Make sure we capture managed exceptions from the Android environment
         AndroidEnvironment.UnhandledExceptionRaiser += AndroidEnvironment_UnhandledExceptionRaiser;
 
-        // Now initialize the Android SDK
+        // Define the configuration for the Android SDK
         SentryAndroidOptions? androidOptions = null;
-        SentryAndroid.Init(AppContext, new JavaLogger(options),
-            new OptionsConfigurationCallback(o =>
+        var configuration = new OptionsConfigurationCallback(o =>
```

Comment:
Note - the only change with this next large block was to extract the `configuration` variable.  Nothing within the block changed.